frontend/src/app/Settings/Users/UsersSideNav.tsx

```diff
@@ -29,7 +29,10 @@ const UsersSideNav: React.FC<Props> = ({
   return (
     <div className="display-block users-sidenav">
       <h2 className="users-sidenav-header">Users</h2>
-      <nav className="prime-secondary-nav" aria-label="Tertiary navigation">
+      <nav
+        className="prime-secondary-nav users-secondary-nav"
```

Comment:
(non-blocking) We can do `maxh-tablet-lg overflow-y-scroll` inline for classes and then we don't need to add the `users-secondary-nav` class. This also fills the user list up to the entire container unless we wanted that whitespace

**Before**:
<img width="1917" alt="Screenshot 2023-10-27 at 09 18 52" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/4bdaf5b0-6d7f-46e4-933a-55331d958104">

**After**
<img width="1843" alt="Screenshot 2023-10-27 at 09 16 57" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/b99cadaf-7f55-417c-a894-81d247929148">

Reply:
I actually played with this a little bit and was not able to figure out how to set the height to be adjustable. Nice find @emyl3 !!!

Reply:
I'll merge this and write a follow up PR with that update.  Thanks!